sdcm/sct_config.py

```diff
@@ -1619,6 +1619,12 @@ class SCTConfiguration(dict):
 
         dict(name="run_scylla_doctor", env="SCT_RUN_SCYLLA_DOCTOR", type=boolean,
              help="Run scylla-doctor in artifact tests"),
+
+        dict(name="num_zero_token_nodes", env="SCT_NUM_ZERO_TOKEN_NODES", type=int,
+             help="""Number of zero token nodes"""),
+
+        dict(name="arbitor_dc", env="SCT_ARBITOR_DC", type=int,
```

Comment:
As I proposed, zero token nodes configuration should be similar to n_db_nodes, so we could define number of them per dc.

Reply:
Exactly.